src/store/Controller.cc

```diff
@@ -811,7 +806,7 @@ Store::Controller::syncCollapsed(const sfileno xitIndex)
     if (sharedMemStore && collapsed->mem_obj->memCache.io == MemObject::ioDone) {
         found = true;
         inSync = true;
-        debugs(20, 7, "fully mem-loaded " << *collapsed);
+        debugs(20, 7, "already handled by memory store: " << *collapsed);
```

Comment:
The branch-added anchorToCache() debugging got it right: "memCache.io == MemObject::ioDone" does not mean that the entire object is currently in the local transient memory. It only means that it was fully loaded there at some point in the past. It may be gone now. It may not even be in the shared memory cache anymore. The difference is not important for this code -- we do not want to anchor the same entry multiple times anyway (at least for now, for simplicity sake) -- but debugs() should not mislead the reader.

I think we should keep the two very similar debugs() in similar code in sync even if doing so triggers this out-of-scope polishing change.